### DISCUSSION TITLE
How to comply with OFL terms

### DISCUSSION THREAD

**Discussion by allanbowe** [vmware-clarity/ng-clarity#424] [Q&A] [answered]
@cds/core has clarity-city as a dependency, licensed under OFL-1.1

Per the licence agreement there is the following clause:

> Standard or Modified Versions of the Font Software may be bundled, redistributed and sold with any software, provided that each copy contains the above copyright notice and this license. These can be included either as stand-alone text files, human-readable headers or in the appropriate machine-readable metadata fields within text or binary files as long as those fields can be easily viewed by the user.

As end users who choose to use the Clarity Framework to build applications for end customers (distributing the web app as a compiled bundle, not a repo), how are we supposed to comply with this?

Do we need to build some kind of 'in app' display feature just for the use of this particular font?

**Comment by pdaigle** [comment, marked as answer]
I apologize, but we aren't able to help you with this legal question. If you're concerned about this, you might want to consult a lawyer familiar with open source licensing.
